### DISCUSSION TITLE
V3.2.0 BLE Joystick doesn't work right. Doesn't everyone fucking care?

### DISCUSSION THREAD

**Discussion by zenlab3d** [ExpressLRS/ExpressLRS#2016] [General]
V3.2.0 BLE Joystick doesn't work right (in velocidrone simulator, radiomaster zorro elrs, win10)

V3.0.1 - BLE Joystick works fine

**Comment by robhaswell** [comment]
This is an incredibly rude way to ask for help. Nevertheless, it seems that there are some fixes for BLE joystick in flight: https://github.com/ExpressLRS/ExpressLRS/pull/2050 - perhaps you can add your feedback here.

**Comment by RandMental** [comment]
Rob Haswell, you the is free software, nobody forces you to use it. Get over yourself and show some appreciation. Sent from my Galaxy
-------- Original message --------From: Rob Haswell ***@***.***> Date: 2023/02/10  18:02  (GMT+02:00) To: ExpressLRS/ExpressLRS ***@***.***> Cc: Subscribed ***@***.***> Subject: Re: [ExpressLRS/ExpressLRS] V3.2.0 BLE Joystick doesn't work right. Doesn't everyone fucking care? (Discussion #2016) 
This is an incredibly rude way to ask for help. Nevertheless, it seems that there are some fixes for BLE joystick in flight: #2050 - perhaps you can add your feedback here.

**Reply by robhaswell** [reply]
Did you mean to write this to me?? I feel like the OP is the one who isn't showing appreciation.
